packages/grafana-ui/src/components/TimePicker/TimePicker.tsx

```diff
@@ -11,15 +14,49 @@ import { ClickOutsideWrapper } from '../ClickOutsideWrapper/ClickOutsideWrapper'
 import { isDateTime, DateTime } from '@grafana/data';
 import { rangeUtil } from '@grafana/data';
 import { rawToTimeRange } from './time';
+import { withTheme } from '../../themes/ThemeContext';
 
 // Types
-import { TimeRange, TimeOption, TimeZone, TIME_FORMAT, SelectableValue } from '@grafana/data';
-import { dateMath } from '@grafana/data';
+import { TimeRange, TimeOption, TimeZone, TIME_FORMAT, SelectableValue, dateMath } from '@grafana/data';
+import { GrafanaTheme, GrafanaThemeType } from '../../types/theme';
+import { Themeable } from '../../types';
 
-export interface Props {
+const orangeLight = '#ED5700';
+const orangeDark = '#FF780A';
+
+const getStyles = memoizeOne((theme: GrafanaTheme) => {
+  const orange = theme.type === GrafanaThemeType.Dark ? orangeDark : orangeLight;
```

Comment:
Preferred way is to use `selectThemeVariant` helper:

```ts
import { selectThemeVariant } from @grafana/ui'; // update path to reletive if in grafana/ui, otherwise lint will fail

const colour =  selectThemeVariant( { light: orangeLight, dark: orangeDark }, theme.type);
```

Btw, ain't there any similar colours in the theme so that we can use instead of specifying new oranges here? And, if this colour is about to be used in other contexts, maybe it's time to put it in the theme? @Ijin08 @jessover9000

Reply:
I tried using the existing oranges, but the darkest one was still too light to get high enough contrast on white background. So I changed the two darkest shades of the 5 available oranges in the color picker. Unless @Ijin08 has any reasons against it, we can absolutely put that change in the theme! The updated oranges can be found here: https://www.figma.com/file/I34UgAga9JvqjlxfKYZnve/Components?node-id=340%3A413

Reply:
OK, does it mean that we can update orange in the theme? Now we have only one- which one from the new palette should be used as a replacement?

I don't want to do it in this PR though, just checking. I think it deserves separate PR to validate viability of these new oranges, especially the one with which we gonna replace current one.

Reply:
sure, let's do a separate PR :) it would be the darkest shade of orange.

Reply:
Ihttps://github.com/grafana/grafana/pull/19407